server/users.ts

```diff
@@ -969,21 +933,16 @@ class User extends Chat.MessageContext {
 		Chat.loginfilter(this, null, userType);
 		return true;
 	}
-	/**
-	 * @param {string} name
-	 * @param {boolean} registered
-	 * @param {boolean} isForceRenamed
-	 */
-	forceRename(name, registered, isForceRenamed = false) {
+	forceRename(name: string, registered: boolean, isForceRenamed: boolean = false) {
```

Comment:
```suggestion
	forceRename(name: string, registered: boolean, isForceRenamed = false) {
```

My old conversions probably used the explicit `foo: boolean = false` pattern, but I now try to leave as much implicit as possible (like return values). Basically, my new philosophy is 'only add explicit types when the inferred type isn't going to be the same so that its clear in which cases its required'.

FWIW I usually prefer `isForceRenamed?: boolean` personally for my *new* APIs, which isnt the case here.